app/frontend/styles/_timeline.scss

```diff
@@ -0,0 +1,117 @@
+/* ==========================================================================
```

Comment:
@adamsilver made some significant tweaks to the MoJ timeline and recommended we use the prototype version

Reply:
So this is the stylesheet copied from the prototype. I think that means that we've got your tweaks @adamsilver - if there are any further tweaks I will have to update and after merging we can just keep the copy up to date I guess.

Reply:
(Sorry, badly phrased comment from me — no pointer intended, I saw your note in the commit message)